src/libs/ajax/methods/Methods.ts

```diff
@@ -40,6 +40,11 @@ export const Methods = (signal?: AbortSignal) => ({
         return res.json();
       },
 
+      delete: async () => {
+        const res = await fetchOrchestration(`api/${root}`, _.merge(authOpts(), { signal, method: 'DELETE' }));
```

Comment:
It seems the other methods here directly call Agora. Is there a reason to go through Orchestration for this API?

Reply:
There is! @sam-schu had asked a question in Slack about whether to use Agora or Orch endpoints, and Adam mentioned that Orch is in fact the intended API as opposed to Agora.

Reply:
Firecloud UI makes this API call to Orchestration, and in a [Slack discussion](https://broadinstitute.slack.com/archives/C06NRD09KBR/p1726075469016359) with @aednichols, I was advised that calling Orchestration is preferred to calling services like Agora directly (and there is a chance that Orchestration will modify the request before making the call to Agora).

Reply:
Thank you for the clarification.